dali/pipeline/workspace/sample_workspace.h

```diff
@@ -76,7 +76,9 @@ class DLL_PUBLIC SampleWorkspace : public WorkspaceBase<SampleInputType, SampleO
    * @brief Returns the index of the sample that this workspace stores
    * in the input/output batch.
    */
-  DLL_PUBLIC inline int data_idx() const { return data_idx_; }
+  DLL_PUBLIC inline int data_idx() const override {
+    return data_idx_;
+  }
```

Comment:
Maybe changes like this should not be part of this PR to keep it clean?

Reply:
Ye, but having separate PR for this seems like overkill with all the rebasing and stuff... I'd say you're right, but we tend to do changes like this ;)